src/ggrc/converters/base.py

```diff
@@ -3,6 +3,7 @@
 
 """Base objects for csv file converters."""
 
+from logging import getLogger
```

Comment:
can you please import logging according to our code style and as we usually do (to be honest a bit more often, not usually :) )

```suggestion
import logging
logger = logging.getLogger(__name__)